database/seeder/seeder.go

```
@@ -0,0 +1,7 @@
+package seeder
+
+import "fmt"
+
+func main() {
```

golangci-lint (1.64.8)

func `main` is unused

(unused)